test/unit/core/store/middleware.ts

```diff
@@ -218,4 +267,58 @@ suite('Middleware', ({ expect, spy, stub }) => {
       expect(thunk).to.be.calledWithExactly(state);
     });
   });
+
+  describe('personalizationAnalyzer()', () => {
+    const conf = { a: 1 };
+    const state = { b: 2 };
+
+    it('should pass the action forward unchanged if not in list of relevant actions', () => {
+      const action = { a: 'b', type: 'NOT_VALID_ACTION' };
+      const config = stub(Selectors, 'config').returns(conf);
+      const enabled = stub(ConfigurationAdapter, 'isRealTimeBiasEnabled').returns(true);
+      const next = spy();
+      const getState = spy(() => state);
```

Comment:
we made this a spy but aren't validating that it's been called

Reply:
Fixed these 3.